examples/gno.land/p/demo/users/users.gno

```diff
@@ -0,0 +1,31 @@
+package users
+
+import (
+	"std"
+	"strconv"
+)
+
+//----------------------------------------
+// Types
+
+type User struct {
+	Address std.Address
+	Name    string
+	Profile string
+	Number  int
+	Invites int
+	Inviter std.Address
+}
+
+func (u *User) Render() string {
```

Comment:
This might be a good time to rename this method to `String()` so it gets the advantage of implementing the `fmt.Stringer` interface for when we eventually support it. I do realize it was already named `Render` and you just moved it; I'm just not sure the name is helpful since `Render` is usually used to mark a function that produces text to be served via gnoweb.

Reply:
Done 8ea3f371a2be2b606a9840c6c287467159ae3de2